recipes/plasmapy/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "0.2.0" %}
+
+package:
+  name: plasmapy
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/plasmapy/plasmapy-{{ version }}.tar.gz
+  sha256: 5048e95700274836443b1f7e533622d7ea7f159b93e938991b7edb7fe43eccc6
+  patches:
+    - setup.cfg.patch
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vvv"
+  skip: True    # [py < 36]
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+  host:
+    - python
+    - pip
+    - cython >=0.28
+    - setuptools
+    - numpy >=1.13
+    - scipy >=0.19
+    - astropy >=2.0
+    - colorama >=0.3
```

Comment:
I suspect you don't need `colorama` and `scipy` in `host`.

(`host` is only for dependencies which are needed to run the `setup.py` script)

Reply:
I suspect we wouldn't necessarily need them, but right now they're in `install_requires`, see here: https://github.com/PlasmaPy/PlasmaPy/blob/master/setup.cfg#L12

It's likely overkill as we're [still figuring out the development process (pic related)](https://hips.hearstapps.com/ghk.h-cdn.co/assets/cm/15/11/54ffe5266025c-dog1.jpg?crop=1xw:1.0xh;center,top&resize=5760:*), but we released 0.2.0 with those in install_requires. We'll probably move them to optional in the future, however, for now I'm betting that this test is going to blow up.

Reply:
I completely agree that it's probably a bad idea to have them there.

Reply:
IIRC `setup_requires` corresponds to `host` dependencies and `install_requires` corresponds to `run` dependencies. It clearly failed though so it doesn't really matter.

I'll merge this once it passes (ping me if I forget :wink:).

Reply:
@chrisburr all green across the board! :smile: